compiler/rustc_lint/messages.ftl

```diff
@@ -403,8 +403,9 @@ lint_inner_macro_attribute_unstable = inner macro attributes are unstable
 
 lint_invalid_asm_label_binary = avoid using labels containing only the digits `0` and `1` in inline assembly
     .label = use a different label that doesn't start with `0` or `1`
-    .note = an LLVM bug makes these labels ambiguous with a binary literal number
-    .note = see <https://bugs.llvm.org/show_bug.cgi?id=36144> for more information
+    .help = Start numbering with `2` instead
+    .note = an LLVM bug makes these labels ambiguous with a binary literal number on x86
+    .note = see <https://github.com/llvm/llvm-project/issues/99547> for more information
```

Comment:
Pre-existing, but I don't think this works like this; this line should use a different identifier and be referenced separately in the `derive(LintDiagnostic)` (i.e. `#[note(lint_issue)]` or whatever).

Reply:
Oh, that's probably why my `.note` wasn't showing up either. Thanks, I'll try it